doc/api/process.md

```diff
@@ -637,6 +637,28 @@ process's [`ChildProcess.disconnect()`][].
 If the Node.js process was not spawned with an IPC channel,
 `process.disconnect()` will be `undefined`.
 
+## process.dlopen(module, filename[, flags])
+<!-- YAML
+added: REPLACEME
```

Comment:
```
$ nvm use 0.5
Now using node v0.5.10
$ node -e 'console.log(process.dlopen)'
[Function]
```

My nvm doesn’t go back further than 0.5, but it was definitely there before. The version we should insert here is the first version in which it was available as `process.dlopen`, which I think is 0.1.8, but at those version ranges it probably doesn’t matter much anyway.